### PR TITLE
Implement Mobile menu logic

### PR DESCRIPTION
**Pull request by khitermedachraf**
### For this branch, We have implemented the next features:

1.  When the user clicks (or taps) the hamburger button, the mobile menu appears.
2. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
3. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
4. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-plusplus */
+/* eslint-disable linebreak-style */
+// Implement the logic of the mobile menu
+// the user clicks (or taps) the hamburger button, the mobile menu appears.
```
Comment (elmar8287) [inline app.js]: - [x] Please, consider removing comments  - this is a bad practice. Let us make a code clear 👍 
- [x] Please, also try to modify the styling of your mobile menu. Here on the screenshot, I made some comments to improve, considering the original Figma design:
> You design (with my comments)
Screen Shot 2022-04-05 at 20 51 28

> Figma design
Screen Shot 2022-04-05 at 20 53 46

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:

- Nice PR description 👍 
- Nice comments history 👍 
- Nice README file 💯 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (khitermedachraf) [reply app.js]: **The changes are implemented, Thank you for the professional review.**

**Review by AmaduKamara — APPROVED** [review]
# STATUS: Approved 🟢🟢🟢 

Hi @khitermedachraf,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @Christianib003, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that belongs to your coding partner @khitermedachraf. You have to submit the project which is in your repo and contributed by your partner. This is a pair programming project where you have to implement the mobile menu for all the repos in your group.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**